backend/tracim_backend/lib/webdav/middlewares.py

```diff
@@ -23,7 +23,7 @@
 from tracim_backend.lib.webdav.dav_provider import WebdavTracimContext
 from tracim_backend.models.auth import AuthType
 from tracim_backend.models.setup_models import get_engine
-from tracim_backend.models.setup_models import get_scoped_session_factory
+from tracim_backend.models.setup_models import get_session_factory
```

Comment:
Because scoped session was the problem here : this creates a session *by thread* that was re-used when the same thread handled another request. As we pass the session object around and the pyramid application behavior is to close and discard session after each request, I did the same here (see the sql alchemy doc https://docs.sqlalchemy.org/en/13/orm/contextual.html?highlight=scoped%20session).

Reply:
ok